packages/cockroach/extra/iam-database-restore

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
```

Comment:
If this file was copied from EE version, then there is a known bug - https://jira.mesosphere.com/browse/DCOS-42407

Reply:
@gpaul can you comment on this? :)

Reply:
Addressed here as well as in https://github.com/mesosphere/dcos-enterprise/pull/3690.

Reply:
(updated https://jira.mesosphere.com/browse/DCOS-42407 correspondingly)